File: practical/week1/section2/MonthlyExpenditureC.c

#include <stdio.h>

int main()
{
    float foodExpenses = 150.0;
    float leisureExpenses = 40.0;
    float clothesExpenses = 100.5;
    float accomodationExpenses = 200.0;
    float travelExpenses = 300.0;

    float total;
    total = foodExpenses + leisureExpenses + clothesExpenses + accomodationExpenses + travelExpenses;

    printf("The total spent this month is £%.1f", total);
    return 0;
}

File: practical/week6/MonthlyExpenditureC.py
# Print monthly expenditure
# Practical 6, Part 2 (a)
# author your name

foodExpenses = float(input("\n Enter food expenses: "))
leisureExpenses = float(input("\n Enter leisure expenses: "))
accommodationExpenses = int(input("\n Enter accomodation expenses: "))
travelExpenses = int(input("\n Enter travel expenses: "))
clothesExpenses = float(input("\n Enter clothes expenses: "))
totalSpent = 0.0

totalSpent = foodExpenses + leisureExpenses + clothesExpenses + accommodationExpenses + travelExpenses

print("The total expenditure this month was: %f" %(totalSpent))
